packages/tests/integration/src/reactiveHints.test.ts

```diff
@@ -1,116 +1,151 @@
-import { Author, Book, BookReview, Comment, Image, Tag } from "@src/entities";
-import { LoadHint, Loaded, Reacted, ReactiveHint, getMetadata, reverseReactiveHint } from "joist-orm";
-import { convertToLoadHint } from "joist-orm/build/reactiveHints";
+import { Author, Book, BookReview, Comment, Critic } from "@src/entities";
+import {
+  Entity,
+  LoadHint,
+  Loaded,
+  MaybeAbstractEntityConstructor,
+  Reacted,
+  ReactiveHint,
+  getMetadata,
+  reverseReactiveHint,
+} from "joist-orm";
+import { ReactiveTarget, convertToLoadHint } from "joist-orm/build/reactiveHints";
 
 const am = getMetadata(Author);
 
 describe("reactiveHints", () => {
   it("can do immediate primitive field names", () => {
-    expect(reverseReactiveHint(Author, Author, "firstName")).toEqual([
-      { entity: Author, fields: ["firstName"], path: [] },
-    ]);
+    expect(reverse(Author, Author, "firstName")).toEqual([{ entity: "Author", fields: ["firstName"], path: [] }]);
```

Comment:
I added a helper method to turn `entity` into a string name, to stop blowing up Jest diffs on the entity constructors.